src/gt4py/next/program_processors/runners/dace_fieldview/gtir_sdfg.py

```diff
@@ -749,7 +808,9 @@ def construct_output_for_nested_sdfg(
                 outer_data = lambda_arg_nodes[inner_data.dc_node.data]
             else:
                 outer_node = head_state.add_access(inner_data.dc_node.data)
-                outer_data = gtir_builtin_translators.FieldopData(outer_node, inner_data.gt_type)
+                outer_data = gtir_builtin_translators.FieldopData(
+                    outer_node, inner_data.gt_type, inner_data.offset
```

Comment:
I am not sure if this is possible, so but in `C++` I would have created a constructor that would unpack `inner_data`, on its own, i.e. 
```C++
FieldopData(
        const OuterNode_t&  outer_node,
        const InnerData_t&   inner_data):
 FieldopData(outer_node, inner_data.gt_type, inner_data.offset)
{}
```
Do you see a way to incorporate something like this?